utf/src/main/kotlin/com/github/softwareAnalysisTeam/unitTestFuzzer/fuzzers/JQFZestFuzzer.kt

```diff
@@ -0,0 +1,293 @@
+package com.github.softwareAnalysisTeam.unitTestFuzzer.fuzzers
```

Comment:
Investigate more on trying to get values of fuzzer without launching zest. Seems like there is no such way described in documentation